infra/terraform/modules/control_panel_api/role.tf

```diff
@@ -148,6 +148,31 @@ resource "aws_iam_policy" "control_panel_api" {
         "arn:aws:iam::${var.account_id}:*"
       ]
     },
+    {
+      "Sid": "CanCreateAndDeleteSSMParameters",
+      "Effect": "Allow",
+      "Action": [
+        "ssm:PutParameter",
+        "ssm:DeleteParameter",
+        "ssm:GetParameterHistory",
+        "ssm:GetParametersByPath",
```

Comment:
Do we need `GetParameterHistory`? And do we not need `GetParameter`?

Reply:
I would've thought cpanel should only set them and not be able to retrieve
the value

Reply:
Yeah, there was something about listing in boto which required history. But no, we don't want to get the parameter in control panel